clients/admin-ui/src/features/privacy-requests/privacy-requests.slice.ts

```diff
@@ -440,10 +460,26 @@ export const {
   useCreateStorageMutation,
   useCreateStorageSecretsMutation,
   useCreateConfigurationSettingsMutation,
+  useGetConfigurationSettingsQuery,
   useGetMessagingConfigurationDetailsQuery,
   useGetActiveMessagingProviderQuery,
   useGetActiveStorageQuery,
   useCreateMessagingConfigurationMutation,
   useCreateMessagingConfigurationSecretsMutation,
   useCreateTestConnectionMessageMutation,
 } = privacyRequestApi;
+
+const EMPTY_CORS_DOMAINS = { domains: [] } as { domains: string[] };
+export const selectCORSDomains = () =>
+  createSelector(
+    [
+      (state) => state,
+      privacyRequestApi.endpoints.getConfigurationSettings.select(),
+    ],
+    (state, { data }) => {
+      // the type checker was having a really hard time with `data`
+      const properlyTypedData = data as ApplicationConfig;
+      const domains = properlyTypedData?.security?.cors_origins;
+      return domains ? { domains } : EMPTY_CORS_DOMAINS;
+    }
+  );
```

Comment:
is there a particular reason why this is in the privacy requests slice file? it probably works just fine here, but seems a little strange to me

Reply:
I found it strange too. The patch route was already here so I added the get route alongside it. I think it was put there originally because it was used for configuring something related to privacy requests